shell.nix

```diff
@@ -19,6 +19,7 @@ pkgs.mkShell {
     cacert
     cargo-udeps
     clang
+    niv
```

Comment:
nit: might suggest leave niv out, you don't need to enter this nix-shell, simple `nix-shell -p niv`

Reply:
Sure..